rclcpp_lifecycle/test/test_lifecycle_node.cpp

```diff
@@ -447,146 +447,6 @@ TEST_F(TestDefaultStateMachine, bad_mood) {
   EXPECT_EQ(1u, test_node->number_of_callbacks);
 }
 
-
-TEST_F(TestDefaultStateMachine, shutdown_from_each_primary_state) {
```

Comment:
this is test enhancement, i will re-submit this cosmetic after revert as follow-up.